admin-manual/maintenance/common-atom-queries.rst

```diff
@@ -0,0 +1,2000 @@
+
+This will return something like the following: 
+
+.. code-block:: mysql
+
+   +---------+--------------+
+   | id      | name         |
+   +---------+--------------+
+   |     193 | Sous-fonds   |
+   |     194 | Collection   |
+   |     195 | Series       |
+   |     196 | Subseries    |
+   |     197 | File         |
+   |     198 | Item         |
+   |   69884 | Record group |
+   | 2000004 | Part         |
+   | 2002760 | Fonds        |
+   +---------+--------------+
+   9 rows in set (0.01 sec)
+
+
+The example above uses English as the target culture. Swap ``en`` in the above
+query example for any other ISO 639-1 two-letter language code as needed. 
+
+If we don't already have the Taxonomy ID but we know the name of the taxonomy, 
+we can nest a taxonomy ID lookup in our query. The following example uses the 
+"Genre" taxonomy as an example: 
+
+.. code-block:: mysql
+
+   SELECT term.id, term_i18n.name FROM term LEFT JOIN term_i18n ON (term.id = term_i18n.id) WHERE term.taxonomy_id = (SELECT id FROM taxonomy_i18n WHERE culture = 'en' AND name = 'Genre') AND term_i18n.culture = 'en';
+
+Finally, we can also look up the individual ID of a specific term if we know 
+the term's name - though for this, we'll need the ID of the related taxonomy. See
+the section above for instructions on how to use SQL to fetch a taxonomy's ID: 
+
+* :ref:`sql-taxonomy-name-id`
+
+In the following example, we'll fetch the ID of the "Item" term, 
+found in the Levels of description taxonomy. Assuming we have discovered that the
+ID for the Levels of description taxonomy is 34, we can used the following to 
+return the ID of the term "Item":
+
+.. code-block:: mysql
+
+   SELECT t.id FROM term t INNER JOIN term_i18n ti ON t.id=ti.id WHERE name='Item' AND t.taxonomy_id=34 AND ti.culture='en';
+
+:ref:`Back to the top <common-atom-queries>` 
+
+.. _sql-change-taxonomy-name:
+
+Change the name of a taxonomy
+-----------------------------
+
+There is no way in the user interface settings to edit the installed name of a 
+:term:`taxonomy` - nor is there any way to add a translation to the default 
+English installation name, if one is not already loaded. However, we can use SQL 
+to edit the name of a taxonomy. 
+
+For example, the "Material Type" taxonomy is used to manage 
+:term:`terms <term>` associated with the Canadian :ref:`RAD <rad-template>` 
+standard template's "General material designation" :term:`drop-down menu` 
+options.  If we wanted to rename this taxonomy in English to 
+"General Material Designation", we first need to know the ID of the target 
+taxonomy. See above: 
+
+* :ref:`sql-taxonomy-name-id` 
+
+In the following example, we've learned that the Id of the Material Type 
+taxonomy is 50, and now we want to update the English title: 
+
+.. code-block:: mysql
+
+   UPDATE taxonomy_i18n SET name='General Material Designation' WHERE culture='en' AND id=50;
+
+**Adding taxonomy name translations**
```

Comment:
Can't this be made through the GUI easily? We have to be careful with the locked taxonomies and protected terms with this kind of queries.

Reply:
Actually, i don't think there's anyway to translate a taxonomy name via the GUI. In the translation bar, it just shows up as `%1%` for example. 

I'll try to add a note about Locked taxonomies and protected terms